frontend/elements/src/components/error/styles.sass

```diff
@@ -8,14 +8,16 @@
   color: variables.$error-color
   background: variables.$background-color
 
-  padding: .25rem
+  padding: .50rem 1.5rem .50rem .50rem 
```

Comment:
Is it intended for the padding on the right to be three times the value of the other sides?
<img width="233" alt="Screenshot 2023-11-01 at 16 08 09" src="https://github.com/teamhanko/hanko/assets/56024829/c35bac7a-35b5-4c82-b8b2-0306daa1cba4">

Reply:
It is intended, That way when the error message is two lines it has a better visual, the text will break at different point.I am trying to bring it closer to the designs

Reply:
Where the text breaks depends on many things, such as the component width, the font size and family, the language, and also there is more than one message, that can be displayed inside the box. So I don't see the point to randomly change the padding just to address one specific case.

Reply:
<img width="451" alt="Screenshot 2023-11-06 at 13 03 00" src="https://github.com/teamhanko/hanko/assets/100808713/65eb57e1-8df2-4e09-a474-549049349759">

This error message is the first interaction with most of the new Hanko users due to an App URL misconfiguration, testing how to make it look better with the default styling we provide can be perceived as random. With your guidance I will find the right values and target the right parts to provide a better display of the error messages.

Reply:
```suggestion
  padding: .5rem
```